src/casp/services/api/services/cell_operations_service.py

```diff
@@ -383,23 +359,22 @@ async def annotate_cell_type_ontology_aware_strategy_with_activity_logging(
                 logger.error(f"Failed to log user activity: {e2}")
             raise e
 
-    async def annotate_cell_type_ontology_aware_strategy(
-        self, user: models.User, file: t.BinaryIO, model_name: str, prune_threshold: float, weighting_prefactor: float
+    async def __annotate_cell_type_ontology_aware_strategy(
```

Comment:
Why do you want to make it private? I think this method should be public based on what it does. Or maybe I am not seeing something, could you please explain?

Reply:
Because this doesn't get (and probably should be) called from outside the class.  This just gets called from `annotate_cell_type_ontology_aware_strategy_with_activity_logging` which is the only thing that should be calling this I think.  Does that make sense?

Reply:
Oh, I forgot about those methods. Got it, makes sense then.